aidial_adapter_bedrock/llm/consumer.py

```diff
@@ -42,23 +42,36 @@ def add_usage(self, usage: TokenUsage):
     def set_discarded_messages(self, discarded_messages: List[int]):
         pass
 
+    @abstractmethod
+    def create_function_tool_call(self, tool_call: ToolCall):
+        pass
+
 
 class ChoiceConsumer(Consumer):
     usage: TokenUsage
     choice: Choice
     discarded_messages: Optional[List[int]]
-    tools_emulator: ToolsEmulator
+    tools_emulator: Optional[ToolsEmulator]
 
-    def __init__(self, tools_emulator: ToolsEmulator, choice: Choice):
+    def __init__(
+        self,
+        choice: Choice,
```

Comment:
nit: remote trailing comma to compress 4 lines into one

Reply:
Fixed